libraries/Microsoft.Bot.Builder/Inspection/InspectionActivityExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder
+{
+    public static class InspectionActivityExtensions
+    {
+        private static JsonSerializerSettings jsonSerializerSettings = new JsonSerializerSettings() { NullValueHandling = NullValueHandling.Ignore };
+
+        public static IEnumerable<Activity> Clone(this List<Activity> activities)
+        {
+            return activities.Select(activity => activity.Clone());
+        }
+
+        public static Activity Clone(this Activity activity)
+        {
```

Comment:
I thought Activity already had a Clone() on it that did this. Consider adding it there, as that seems a more logical place.

Reply:
no cloning now!!! :-)